suite/tests/client-interface/drreg-test.dll.c

```diff
@@ -369,6 +369,28 @@ check_const_ne(ptr_int_t reg, ptr_int_t val)
     CHECK(reg != val, "register value not preserved");
 }
 
+static void
+empty_clean_call()
+{
+    /* TODO: fix memory leak when the following statement is removed and this
```

Comment:
Not sure if this is a known issue. If not, I'll file a new one.

Reply:
Best to put an issue number by any TODO comment.